### ISSUE TITLE
[a11y][Lighthouse] Date, name and address question text not read by screen reader

### ISSUE THREAD

**Issue by melanie-exygy** [civiform/civiform#12907]
**Describe the bug**
When filling out an application using the screen reader, residents don't hear the question text for date, name and address questions.  This happens when they are using the tab key to navigate or using form navigation keys (see cheat sheet link below) such as "F".

This happens with NVDA or JAWS screen readers, not with VoiceOver.

**WCAG Success Criterion**
[Success Criterion 3.3.2](https://www.w3.org/TR/WCAG21/#labels-or-instructions)

**To Reproduce**

1. As a resident, start an application for the Comprehensive Sample Program or any program with a date, name and address question.
2. Turn the NVDA or JAWS screen reader on.
3. Tab to the input field on the date question.
4. Listen to the screen reader and notice that it never reads the question text, such as "What is your birth date?"
5. Do the same with the other two question types.

**Expected behavior**
When you navigate to the first field of the date question, the question text should be read by the screen reader.

**Screenshots**

https://github.com/user-attachments/assets/2a6a0923-5e65-41c6-a490-994e3535d6d3

**Desktop:**

- OS: Windows
- Browser: Chrome
- Screen Reader: JAWS, NVDA

**Additional context**
Here are the NVDA form navigation keys:  https://dequeuniversity.com/screenreaders/nvda-keyboard-shortcuts#nvda-forms.

Interestingly, the help text is read.  We may want to use the same method to programmatically associate the label.
